src/libraries/System.Formats.Asn1/src/System/Formats/Asn1/AsnWriter.cs

```diff
@@ -22,8 +22,11 @@ public sealed partial class AsnWriter
         private Stack<StackFrame>? _nestingStack;
 
         /// <summary>
-        ///   The <see cref="AsnEncodingRules"/> in use by this writer.
+        ///   Gets the encoding rules in use by this writer.
```

Comment:
There are some other methods in this file (e.g., `Reverse` on line 596) that won't be needed once targeting netstandard2.1+ or net5.0+. Are we keeping track of these anywhere?